apps/wallet/src/features/Send/InputTransaction/index.tsx

```diff
@@ -30,6 +34,16 @@ const InputTransaction: FC<Props> = ({ navigator }) => {
 		}
 	};
 
+	useEffect(() => {
+		if (collectible) {
+			txActions.update({ type: 'Collectible' });
+			const collection = collections.find(
+				(ele) => ele._id === collectible.collectionId,
+			);
+			txActions.update({ collection });
+		}
+	}, [collectible]);
```

Comment:
I suggest using method `getCollectionByIdFromStorage` in `utils/storage/collectible.ts` to make code more transparent

Reply:
please check the latest change